app/templates/index.html

```diff
@@ -20,6 +20,47 @@ <h2><span style="color:red;">CONP</span> Portal</h2>
 
 <!-- Content Block -->
 {% block appcontent %}
+
+<!-- Modal -->
+<div class="modal fade" id="termsModal" tabindex="-1" role="dialog" aria-labelledby="exampleModalCenterTitle"
+  aria-hidden="true">
+  <div class="modal-dialog modal-dialog-centered" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h5 class="modal-title" id="exampleModalLongTitle">Terms of Use</h5>
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+      <div class="modal-body">
+        Terms go here
+      </div>
+      <div class="modal-footer">
+        <button type="button" class="btn btn-primary" id="btnAcceptTerms">Accept</button>
+      </div>
+    </div>
+  </div>
+</div>
+
+<!-- Trigger modal -->
+{% if showTerms %}
+<script type="text/javascript">
+  $('#termsModal').modal('show')
+</script>
+{% endif %}
+
+<script>
+  $(function () {
+    $('#btnAcceptTerms').on('click', function (event) {
+      document.cookie = "termsAccepted=true";
```

Comment:
it would be nice to add expire= so the cookie is kept after the session
I will create an issue